schemas/data.schema.json

```diff
@@ -273,117 +179,179 @@
+              "type": "string"
             },
             "safari": {
-              "$ref": "#/definitions/BrowserData"
+              "type": "string"
             },
             "safari_ios": {
-              "$ref": "#/definitions/BrowserData"
+              "type": "string"
             }
           },
-          "required": [
-            "chrome",
-            "chrome_android",
-            "edge",
-            "firefox",
-            "firefox_android",
-            "safari",
-            "safari_ios"
-          ],
-          "type": "object"
-        },
-        "features": {
+          "additionalProperties": false
+        }
+      },
+      "required": ["baseline", "support"]
+    },
+    "StatusHeadline": {
+      "type": "object",
+      "allOf": [{ "$ref": "#/definitions/Status" }],
+      "properties": {
+        "baseline": {},
+        "baseline_high_date": {},
+        "baseline_low_date": {},
+        "support": {},
+        "by_compat_key": {
+          "description": "Statuses for each key in the feature's compat_features list, if applicable. Not available to the npm release of web-features.",
+          "type": "object",
           "additionalProperties": {
-            "$ref": "#/definitions/FeatureData"
-          },
-          "description": "Feature identifiers and data",
-          "type": "object"
+            "$ref": "#/definitions/Status"
+          }
+        }
+      },
+      "required": ["baseline", "support"],
+      "additionalProperties": false
+    },
+    "SnapshotData": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Short name",
+          "type": "string"
         },
-        "groups": {
-          "additionalProperties": {
-            "$ref": "#/definitions/GroupData"
-          },
-          "description": "Group identifiers and data",
-          "type": "object"
+        "spec": {
+          "description": "Specification",
+          "$ref": "#/definitions/URL"
+        }
+      },
+      "required": ["name", "spec"],
+      "additionalProperties": false
+    },
+    "StringOrStrings": {
+      "oneOf": [
+        {
+          "type": "string"
         },
-        "snapshots": {
-          "additionalProperties": {
-            "$ref": "#/definitions/SnapshotData"
+        {
+          "type": "array",
+          "items": {
+            "type": "string"
           },
-          "description": "Snapshot identifiers and data",
-          "type": "object"
+          "minItems": 2
         }
+      ]
+    },
+    "Strings": {
+      "type": "array",
+      "items": {
+        "type": "string"
       },
-      "required": [
-        "browsers",
-        "features",
-        "groups",
-        "snapshots"
-      ],
-      "type": "object"
+      "minItems": 1
+    },
+    "URL": {
+      "type": "string"
+    },
+    "URLs": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "minItems": 1
+    },
+    "URLOrURLs": {
```

Comment:
Nit: Currently this is the same as StringOrStrings. I think you meant to add the `format` of the string? Otherwise, could you simplify to use StringOrStrings.